sympy/functions/special/tests/test_gamma_functions.py

```diff
@@ -442,3 +443,46 @@ def test_issue_14450():
 def test_issue_14528():
     k = Symbol('k', integer=True, nonpositive=True)
     assert isinstance(gamma(k), gamma)
+
+def test_multivariate_gamma():
+    from sympy import Product
+    p = Symbol('p')
+    _k = Dummy('_k')
+
+    assert str(multivariate_gamma(x, p)) == ("pi**(p*(p - 1)/4)*Product(gamma(-_k/2 + x + 1/2), "
```

Comment:
Can `dummy_eq` be used here?

Reply:
`dummy_eq` does not seems to work. I have used `str` method elsewhere also.

Reply:
This is because you've replaced the symbol (see comments below)